src/components/Sidebar.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+const Sidebar = ({ categories, event }) => {
+  return (
+    <div className="Sidebar">
+      {categories.map((category, index) => {
+        if (category.selected) {
+          return (
+            <div className="Selected-sidebar-element" onClick={() => event(category.id)} key={category.id}>
+              {category.name}
+            </div>
```

Comment:
A more elegant way is to use a dynamic `className` so we don't repeat this codeblock for the selected and not selected categories